src/components/2022/related-article.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Grid from "./grid"
+import GridCell from "./grid-cell"
+
+const RelatedArticle = ({ image, title, description, href }) => {
+  return (
+    <a className="cmp-related-reading__link" href={href}>
+      <Grid gap="lg" gapColMD="1xl">
+        <GridCell spanSM={2} spanMD={2}>
+          <img src={image} alt="" className="cmp-related-reading__image" />
+        </GridCell>
+        <GridCell spanSM={4} startSM={3} spanMD={5} startMD={3} spanLG={6}>
+          <h4 className="cmp-type-body-xlarge util-margin-all-none util-weight-regular">{title}<svg className="cmp-related-reading__arrow" width="14" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M11.703 6.36H1.058v9.16" stroke="currentColor" stroke-width="1.5"/><path d="m7.383 1.52 4.675 4.838-4.675 4.84" stroke="currentColor" stroke-width="1.5"/></svg></h4>
```

Comment:
stick an aria-hidden on your svg

Reply:
addressed

Reply:
I like the foundry images in white. Very Louise Nevelson.
<img width="566" alt="Screen Shot 2022-06-14 at 11 32 37 AM" src="https://user-images.githubusercontent.com/25614178/173617443-b5d50764-1339-4eeb-a265-662306e24ccc.png">

Reply:
ohhhh for real!